common/cultures/culture_titanforged.txt

```diff
@@ -481,6 +481,41 @@ titanforged_group = {
 		mat_grm_name_chance = 40
 		mother_name_chance = 5
 
+		# Modifiers
+		modifier = default_culture_modifier
+	}
+	mogu = {
```

Comment:
What about the mogu culture being in their own group. Their words and letters barely connected to the Titan language. Pandaren, jinyu, saurok, etc. will be in the mogu group.
https://wow.gamepedia.com/Mogu_(language)#Speculation

Reply:
mogu and saurok constitute of one culture now